app/views/assignments/_assignment_roster_list.html.erb

```diff
@@ -0,0 +1,5 @@
+<% roster_entries.each do |entry| %>
```

Comment:
Can we feature flag the entire search/filter feature? That way, we can slowly roll it out and monitor performance and feedback, and it'll be easy to rollback if we need to.

Reply:
Yes we can definitely do that

Reply:
Yes! I was going to ask: how specifically do I do that. Is it just a simple method conditional?

Reply:
You can look into https://github.com/education/classroom/blob/8b09ce135f92eb6ea3753df104e68ec847dfdd95/app/controllers/application_controller/feature_flags_dependency.rb which defines our flipper helpers.

More info: https://github.com/jnunemaker/flipper

I believe rosters are behind a feature flag right now, you can also use those as an example.

Reply:
Just added the feature flag